.github/workflows/ci.yml

```diff
@@ -83,8 +83,7 @@ jobs:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
       - uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
         with:
-          node-version: '*'
-          check-latest: true
+          node-version: 'lts/*'
```

Comment:
FWIW these were all set to latest because they would benefit from being as fast as possible, but, I guess it's too risky.